.github/workflows/nv-flash-attn.yml

```diff
@@ -0,0 +1,52 @@
+name: nv-flash-attn
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
```

Comment:
Should probably add the test path as well as the fpdt_later.py file (or the whole `deepspeed/sequence` folder as path triggers too.